Source/Diagnostics/FlushFormats/FlushFormatOpenPMD.cpp

```diff
@@ -66,7 +66,6 @@ FlushFormatOpenPMD::FlushFormatOpenPMD (const std::string& diag_name)
   std::string const prefix = diag_name + ".adios2_operator.parameters";
   ParmParse pp;
   auto entr = pp.getEntries(prefix);
```

Comment:
Let's keep this one for a visual split between inputs & option building?
```suggestion
  auto entr = pp.getEntries(prefix);

```